common/index.ts

```diff
@@ -145,4 +145,22 @@ export const LAYER_ICON_TYPE_MAP: { [key: string]: string } = {
 
 export enum TOOLTIP_STATE {
   DISPLAY_FEATURES = 'DISPLAY_FEATURES',
+  FILTER_DRAW_SHAPE = 'FILTER_DRAW_SHAPE',
 }
+
+export enum FILTER_DRAW_MODE {
+  NONE = 'none',
+  POLYGON = 'polygon',
+}
```

Comment:
Could you provide some words on what's the NONE and POLYGON mean?

Reply:
Ack